src/map_homework/model/Student.java

```diff
@@ -0,0 +1,65 @@
+package map_homework.model;
+
+import java.util.*;
+
+public class Student {
+    private String firstName;
+    private String lastName;
+    private int age;
+    private String phoneNumber;
+    private String faculty;
```

Comment:
How can we make sure the faculty is always correctly inserted? e.g. is not Myth instead of Math

Reply:
We can use enums for it, soon I will push code with enums used for it